airflow/models.py

```diff
@@ -3505,6 +3505,21 @@ def sync_to_db(dag, owner, sync_time, session=None):
         for subdag in dag.subdags:
             DAG.sync_to_db(subdag, owner, sync_time, session=session)
 
+    @staticmethod
+    @provide_session
+    def deactivate_all_dags(session=None):
+        """
+        Deactivate all the existing dags to be in active.
+
+        :param session:
+        :return:
+        """
+        for dag in session.query(
```

Comment:
NIT: looks like it should fit on one line

Reply:
This may cause the UI to "flicker" between the time where this happens and things get re-activated

Reply:
Hi @mistercrunch , thanks for the comment. I feel the UI flicker should not be that an issue as the scheduler should check all the DAGs pretty quickly? Do you feel this is a valid approach to solve this issue as I am not 100% sure if it will affect or cause any synchronization between scheduler and worker?